gtsfm/two_view_estimator.py

```diff
@@ -86,4 +94,51 @@ def create_computation_graph(
             exact_intrinsics,
         )
 
-        return i2Ri1_graph, i2Ui1_graph, v_corr_idxs_graph
\ No newline at end of file
+        # if we have the expected data, evaluate the computed relative pose
+        if i2Ti1_expected_graph is not None:
+            error_graphs = dask.delayed(compute_metrics)(
+                i2Ri1_graph, i2Ui1_graph, i2Ti1_expected_graph
+            )
+        else:
+            error_graphs = (None, None)
+
+        return (
+            i2Ri1_graph,
+            i2Ui1_graph,
+            v_corr_idxs_graph,
+            error_graphs[0],
+            error_graphs[1],
+        )
+
+
+def compute_metrics(
+    i2Ri1_computed: Optional[Rot3],
+    i2Ui1_computed: Optional[Unit3],
+    i2Ti1_expected: Pose3,
+) -> Tuple[float, float]:
+    """Compute the metrics on relative camera pose.
+
+    Args:
+        i2Ri1_computed: computed relative rotation.
+        i2Ui1_computed: computed relative translation direction.
+        i2Ti1_expected: expected relative pose.
+
+    Returns:
+        Rotation error.
+        Unit translation error.
+    """
+
+    R_error = comp_utils.compute_relative_rotation_angle(
+        i2Ri1_computed, i2Ti1_expected.rotation()
+    )
+
+    U_error = comp_utils.compute_relative_unit_translation_angle(
+        i2Ui1_computed, Unit3(i2Ti1_expected.translation())
+    )
+
+    logging.debug("[Two View Estimator] Relative rotation error %f", R_error)
```

Comment:
prefer to pull get the default logger here with the logger utils. Is it printing out currently as expected when you run it?

Reply:
leaving this for now, as discussed